lib/ansible/modules/windows/win_reboot.py

```diff
@@ -31,6 +31,7 @@
     description:
     - Maximum seconds to wait for shutdown to occur
     - Increase this timeout for very slow hardware, large update applications, etc
+    - This option has been removed since Ansible 2.5 as the win_reboot behaviour has changed
```

Comment:
s/behaviour/behavior/ (IIRC we are standardizing on US English)

Reply:
Booo ! The British could use some support right now... ;-)

Reply:
It's not just the British who spell it this way, only the US seem to have their own way of spelling things :)

Will change it